src/main.rs

```diff
@@ -334,6 +344,16 @@ struct CheckRelease {
     )]
     baseline_rustdoc: Option<PathBuf>,
 
+    /// Assume given semver type and ignore the actual version difference.
```

Comment:
```suggestion
    /// Assume the version change is of the given semver type, overriding the actual change kind.
```
Just a slight wording tweak that hopefully makes this a bit more clear. What do you think?

Reply:
I played around with it for a while, never reaching a state that I was fully happy with, so if you think this is better I'm very open to accept it ;)